app/views/sign_up/index.html.haml

```diff
@@ -10,10 +10,16 @@
         %li updates to registers, the latest available registers, and upcoming registers
         %li tips and guidance on how to get the best out of registers
 
-      = form_for @sign_up_user, url: thank_you_for_signing_up_path() + @params, html: { class: "js--ga-submit" } do |f|
+      - if flash[:alert].present?
+        %div{:class => 'error-summary', :role => 'alert'}
+          %h2{:class => %w(heading-medium error-summary-heading), :id => 'error-summary-heading'}
+            = flash.alert
+          %p!="If the problem persists please contact  #{link_to 'support', support_path}"
+
+      = form_for @sign_up_user, url: sign_up_index_path({from: @register&.slug, method: params[:method]}.compact), html: { class: "js--ga-submit" } do |f|
         .form-group
-          = label_tag("email-address", "Email address", class: "form-label-bold")
-          = email_field_tag("email-address", nil, class: "govuk-input govuk-!-width-one-half", required: true)
+          -# = label_tag(:email, "Email address", class: "form-label-bold")
+          = f.text_field(:email, nil, class: "govuk-input govuk-!-width-one-half", required: true)
```

Comment:
did this change affect the styling? If so @injms needs to take a look at it. Also the commented out line above should be deleted.

Reply:
Ah, yeah it did. @injms are you ok to fix this? The separate labels used to be bolded but I wasn't sure how to accomplish the same thing with `f.text_field`.

We're using the `f` object so that the fields are bound to the model object, and user input gets preserved if we re-render the same page when there's an error.

Reply:
Couldn't find a way of adding a class to the label other than writing a new form helper - which we might have to do in the future for the new design system, but that's a problem for then. 

The effect has been achieved with a bit of CSS. Not ideal, but it will do.